src/PresentationalComponents/NoSystems/NoSystems.js

```diff
@@ -34,25 +33,13 @@ const NoSystems = () => {
                         <StackItem>
                             {intl.formatMessage(messages.noSystemsDescription)}
                         </StackItem>
-                        <StackItem>
-                            <span className='ins-c-no-systems__helper'>
-                                1. {intl.formatMessage(messages.noSystemsInstall)}
-                            </span>
-                            <ClipboardCopy>yum install insights-client</ClipboardCopy>
-                        </StackItem>
-                        <StackItem>
-                            <span className='ins-c-no-systems__helper'>
-                                2. {intl.formatMessage(messages.noSystemsRegister)}
-                            </span>
-                            <ClipboardCopy>insights-client --register</ClipboardCopy>
-                        </StackItem>
                     </Stack>
                 </EmptyStateBody>
                 <Button
                     component="a"
-                    href="https://access.redhat.com/products/red-hat-insights#getstarted"
-                    target="_blank" variant="primary">
-                    {intl.formatMessage(messages.noSystemsLink)}
+                    href="https://cloud.redhat.com/beta/insights/registration"
```

Comment:
OOOOO good point this is only on beta atm 🤔 guess we'll do another pr to update this url once it shows up on stable